packages/daemon/__tests__/integration/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: "3.9"
+
+services:
+  mysql:
+    image: mysql
+    networks:
+      - database
+    environment:
+      MYSQL_ROOT_PASSWORD: hathor
+    ports:
+      - "3380:3306"
+  unvoided_transaction:
+    image: hathornetwork/hathor-core:stable
+    command: [
+      "events_simulator",
+      "--scenario", "UNVOIDED_TRANSACTION",
+      "--seed", "1"
+    ]
+    ports:
+      - "8081:8080"
+  reorg:
+    image: hathornetwork/hathor-core:stable
+    command: [
+      "events_simulator",
+      "--scenario", "REORG",
+      "--seed", "1"
+    ]
+    ports:
+      - "8082:8080"
+  single_chain_blocks_and_transactions:
+    image: hathornetwork/hathor-core:stable
+    command: [
+      "events_simulator",
+      "--scenario", "SINGLE_CHAIN_BLOCKS_AND_TRANSACTIONS",
+      "--seed", "1"
```

Comment:
Shouldn't we let the seeds be random?

Reply:
We could, but then we would have no way to check if the balance is correct as the fullnode's API is not active when using the simulator

It's fixed so we can have a list of addresses and their balances

Reply:
Ok!